earthly/docs/output/kroki/docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+version: "3"
```

Comment:
Why is this in version control? I assume it's an output from the Earthly build.

Reply:
Yes, I culled it.  Accidentally got checked in.